settings/post-process.sql

```diff
@@ -95,3 +95,80 @@ FROM
 WHERE
   osm_bus_points.osm_id = osm_bus_route_members.member_osm_id
 ;
+
+-- Deduplicate linstring
+DROP TABLE IF EXISTS osm_bus_ways;
+CREATE TABLE osm_bus_ways AS
+SELECT
+  rel_ref,
+  diameter,
+  (ST_Dump(geometry)).geom AS geometry
+FROM
+(
+  SELECT
+    rel_ref,
+    max(diameter) AS diameter,
+    ST_LineMerge(ST_Union(geometry)) AS geometry
+  FROM
+  (
+    SELECT
+      member_osm_id,
+      string_agg(DISTINCT rel_ref, '  ') AS rel_ref,
+      max(diameter) AS diameter,
+      max(geometry) AS geometry
+    FROM
+      osm_bus_route_members
+    WHERE
+      member_type = 1 AND -- Way
+      member_role NOT LIKE '%stop%'
```

Comment:
It should not happen, but you can also encounter ways with platform role
![screenshot-2018-4-24 http localhost 2](https://user-images.githubusercontent.com/919962/39207940-9d7da0e8-4801-11e8-8e3f-ead754af9f63.png)
https://www.openstreetmap.org/way/119363692

Reply:
Fixed